src/Microsoft.Data.SqlClient/netcore/src/Microsoft.Data.SqlClient.csproj

```
@@ -344,6 +344,7 @@
     <Compile Include="Microsoft\Data\SqlClient\SqlDiagnosticListener.NetStandard.cs" />
     <Compile Include="Microsoft\Data\SqlClient\SqlDelegatedTransaction.NetStandard.cs" />
     <Compile Include="Microsoft\Data\SqlClient\TdsParser.NetStandard.cs" />
+    <Compile Include="Microsoft\Data\SqlClient\SNI\SNIStreams.Task.cs" />    
```

Applies to `'$(TargetGroup)' == 'netstandard'` :

```suggestion
    <Compile Condition="'$(TargetFramework)' == 'netstandard2.0'" Include="Microsoft\Data\SqlClient\SNI\SNIStreams.Task.cs" />
    <Compile Condition="'$(TargetFramework)' != 'netstandard2.0'" Include="Microsoft\Data\SqlClient\SNI\SNIStreams.ValueTask.cs" />
```

Anything above netstandard2.0 will take in ValueTask code.

---

anything above netstandard 2.0 is netstandard 2.1 or netcore 3.1 or later, so valuetask is fine.